images/registry-replacer/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM centos:7
```

Comment:
Care to bump to 8 given you bumped the applyconfig one?

Reply:
Done